app/models/service_template_ansible_playbook.rb

```diff
@@ -225,6 +228,8 @@ def update_job_templates(name, description, config_info, auth_user)
 
   def validate_update_config_info(options)
     opts = super
+    return unless options.key?(:config_info)
```

Comment:
Should the `return` happen before the call to `super`? Or is the super method modifying `options`?

Reply:
I think not, since Bill said (https://github.com/ManageIQ/manageiq/pull/18981#issue-297767141) to do exactly this.